CHANGELOG.md

```diff
@@ -6,6 +6,30 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+## 3.4.0    
+
+#### General    
+
+* `FEAT`: add notifications ([#1505](https://github.com/camunda/camunda-modeler/issues/1505))
+* `FEAT`: add extension point for ui plugins ([#1490](https://github.com/camunda/camunda-modeler/issues/1490))
+* `FEAT`: add stable api to access workspace configuration fron plug-ins ([#1490](https://github.com/camunda/camunda-modeler/issues/1425))
```

Comment:
"from"

I'd say

"allow access to workspace configurations for plugins and files"

+ Ensure you reference the correct issue number (1425)